ruby/ql/test/query-tests/experimental/CWE-522-DecompressionBombs/DecompressionBombs.expected

```diff
@@ -0,0 +1,114 @@
+edges
```

Comment:
This `.expected` file looks like it needs to be regenerated to account for a change in the `edges` relation format.